salt/modules/x509.py

```diff
@@ -1464,12 +1464,34 @@ def create_certificate(
                 serial_number -= int(serial_number / sys.maxsize) * sys.maxsize
     cert.set_serial_number(serial_number)
 
+    # If a not_before date was set
```

Comment:
Please add some documentation about these two new kwargs

Reply:
Documentation was amended to reflect the 2 new kwargs